### ISSUE TITLE
Data Viewer: consecutive spaces display as 1 space

### ISSUE THREAD

**Issue by jennybc** [rstudio/rstudio#2499]
It seems odd that n consecutive spaces, n > 1, display as if n = 1.

``` r
df <- data.frame(
  x = c("12", "1  2", "1    2", "1      2", "1        2"),
  y = c("34", "3__4", "3____4", "3______4", "3________4"),
  stringsAsFactors = FALSE
)
View(df)
df$x
```

<img width="754" alt="screen shot 2018-03-25 at 7 05 03 pm" src="https://user-images.githubusercontent.com/599454/37883523-d04222e2-305f-11e8-8410-ff884f126521.png">

**Comment by etienne-s** [comment]
It's the same with tabs and line feeds. And leading/trailing spaces, tabs and line feeds are trimmed.

But I believed it was a feature.

**Comment by ronblum** [comment]
Reproduce in Mac Desktop and Linux Server, 1.1.442 and 1.2.472.

**Comment by jmcphers** [comment]
It's always been this way, and it's because the data viewer is an HTML widget based on [DataTables](https://datatables.net/). We could definitely make this work although it wouldn't be as straightforward as it might immediately appear (for example the built-in search is very literally minded so would also need some work to recognize `&nbsp;` as a space).

**Comment by jmcphers** [comment]
@jennybc thanks for reporting this! should be fixed in the latest dailies.

**Comment by dfalty** [comment]
Verified fixed on version 1.2.1090.
